### ISSUE TITLE
Create SimCLR model using Lightly library and PyTorch Lightning 

### ISSUE THREAD

**Issue by Gryuuum** [hits-sdo/hits-sdo-similaritysearch#18]
## Description

We want to create a SimCLR model using the Lightly library and PyTorch Lightning module.  

## Tasks

-[ ] Port SimCLR LightningModule code
-[ ] Configure SimCLR Lightly code to use pre-trained model
-[ ] Train using NTXentLoss()
-[ ] Configure with Weights & Biases logging
-[ ] Evaluate results using a subset or validation set of data
-[ ] Plot TSNE 2D representation of latent space

**Comment by nadia-eecs** [comment]
[PR#31](https://github.com/hits-sdo/hits-sdo-similaritysearch/pull/31)  closes #18
